server/schedule/event_handler.go

```diff
@@ -35,6 +35,21 @@ func (open *OpenEvent) ToHeartbeatResp() *metaservicepb.NodeHeartbeatResponse {
 	}
 }
 
+type CloseEvent struct {
+	ShardIDs []uint32
+}
+
+func (open *CloseEvent) ToHeartbeatResp() *metaservicepb.NodeHeartbeatResponse {
```

Comment:
Actually, in the following plan, the close/open command will not be delivered by the heartbeat stream, so for now, there is no need to take considerations for this.